docs/chainbridge/setup/setup-erc20-transfer.md

```diff
@@ -0,0 +1,148 @@
+---
+id: setup-erc20-transfer
+title: ERC20 Token Transfer
+---
+
+So far, we've set up bridge to exchange assets/data between Polygon PoS and Polygon SDK chain. This section guides you a way how to set up ERC20 bridge and send tokens to another chain.
+
+## Step 1: Register resource ID
+
+Firstly, you will register a resource ID which associates resources in a cross-chain environment. A Resource ID is 32 bytes value which must be different from other resources but the same on every chain. The Resource IDs are arbitrary, but they may have the chain ID of the home chain in the last byte.
+
+To register resource ID, you can use `cb-sol-cli bridge register-resource` command. You will need to give the private key of `admin` account.
+
+```bash
+# For Polygon PoS chain
+$ cb-sol-cli bridge register-resource \
+  --url https://rpc-mumbai.matic.today \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  # Set Resource ID for ERC20
+  --resourceId "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00" \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --handler "[ERC20_HANDLER_CONTRACT_ADDRESS]" \
+  --targetContract "[ERC20_CONTRACT_ADDRESS]"
+
+# For Polygon SDK chain
+$ cb-sol-cli bridge register-resource \
+  --url http://localhost:10002 \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  # Set Resource ID for ERC20
+  --resourceId "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00" \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --handler "[ERC20_HANDLER_CONTRACT_ADDRESS]" \
+  --targetContract "[ERC20_CONTRACT_ADDRESS]"
+```
+
+## (Optional) Make contracts mintable/burnable
+
+When you send ERC20 tokens between chains, tokens are processed by either way as follows:
+
+(1) Lock/Release mode
```

Comment:
Provide some information on the Lock/Release mode, as to how the liquidity is provided on the "non-home" chain.

Reply:
Fixed at 5756c760532fdfa7da42c47aad4dc893d1e4074e

Reply:
@lazartravica In the "non-home" chain, the Handler contract has to have enough amount of Tokens beforehand if the contract is set to `Lock/Release` mode. The tokens won't be minted automatically and the Handler contract can transfer the same amount of the tokens in the Handler contract.